ansible-roles/roles/secure-boot/tasks/common.yml

```diff
@@ -0,0 +1,37 @@
+---
+- name: Install dependency packages
```

Comment:
While developing we named it `common` as Ubuntu was only OS we tested this on. It will not run on `Windows` because of https://github.com/Dasharo/open-source-firmware-validation/blob/uefi-sb-release/lib/ansible.robot#L16.

We could name it after an OS but then I think we would need some env var to say on which OS given test setup is ran. I did not want to add another variable set on cmdline line but also did not have another idea for this.

If there was a variable for this, to simplify whole ansible KWD we would also need to use that variable with `Boot System Or From Connected Disk` KWD https://github.com/Dasharo/open-source-firmware-validation/blob/uefi-sb-release/lib/ansible.robot#L18